### DISCUSSION TITLE
Wondering if you guys would want to have a `Citation.cff` file for citing the usage of marimo?

### DISCUSSION THREAD

**Discussion by Haleshot** [marimo-team/marimo#1743] [General]
After going through the repository of a few plotting/data visualization frameworks (in relation to a PR related issue in marimo), I came across a `Citation.cff` file in ArviZ framework repo on GitHub. Both marimo and ArviZ are licensed under `Apache-2.0`; was just wondering if you guys would like to have a file which helps in citing the usage of marimo (consequently resulting in a Used by category in GitHub as dependents).

Link to ArviZ citation file - https://github.com/arviz-devs/arviz/blob/main/CITATION.cff

This is obviously something which you core maintainers would want to decide on; just wanted to post it here. Do let me know if I should close the discussion. Thanks.

**Comment by akshayka** [comment]
@Haleshot sure, that's a good idea! Would you like to add one to the repo in a PR?

**Reply by Haleshot** [reply]
> @Haleshot sure, that's a good idea! Would you like to add one to the repo in a PR?

That works for me! I will refer to the following references:
1) https://citation-file-format.github.io/
2) https://github.com/arviz-devs/arviz/blob/main/CITATION.cff
3) https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#about-citation-files
4) https://github.com/citation-file-format/ruby-cff

**Comment by Haleshot** [comment]
Using the https://citation-file-format.github.io/cff-initializer-javascript site and exploring it's options. It seems most of the common citations file I come across require either a DOI or a orcid number (unique to recognize). I can get a PR running soon.
